controllers/hyperconverged/hyperconverged_controller.go

```diff
@@ -1470,3 +1476,60 @@ func checkFinalizers(req *common.HcoRequest) bool {
 	}
 	return false
 }
+
+type Kubeletconfig struct {
```

Comment:
this holds that structure to parse the API response for `/nodes/proxy/configz`, maybe this is not the best place for it, any suggestions?